### DISCUSSION TITLE
[Pi] Workaround, Mono crashing when using ini setting

### DISCUSSION THREAD

**Discussion by HaroldTaylor** [ORelio/Minecraft-Console-Client#1869] [Q&A] [answered]
For some reason, it always crashes for me when it uses the ini settings. I've got a workaround by making a little bash script that deletes the ini setting and just lunches the exe the username password and server ip; however, this only works for mojang accounts as I don't think you can change the login type on the command line, if you can please let me know 😋

**Comment by ORelio** [comment, marked as answer]
I'm not sure about your issue with INI file, but as per [command-line usage](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#command-line-usage), `--type=microsoft` should do what you want.
